test/pages/chat/events/message/message_content_builder_mixin_test.dart

```diff
@@ -16,6 +16,8 @@ import 'package:matrix/matrix.dart';
 import 'package:matrix_api_lite/fake_matrix_api.dart';
 import 'package:flutter_gen/gen_l10n/l10n.dart';
 
+import '../../../../utils/shared_mocks.dart';
```

Comment:
Pls import specific path

Reply:
Not possible on test files because it's not in the app folder (I mean `lib`). So we have to use relative path